website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
Isn't the work instead attributed via the ReadMe & the Git commits themselves? We have line-by-line copyright built in, and I have personally found these copyrights difficult to maintain & dirty the diff. 

The copyright line is strictly for legal assertion of copyright. Adding a name to it after the company is ambiguously representing that the person also holds a copyright to that code, which is False.